### PR TITLE
Draft PGXN v2 update

### PR DESCRIPTION
**Pull request by theory**
- [x] The `image: ` field in frontmatter of blogs should always be a `png` (`svg` will not work for social previews)
- [x] You must also duplicate the asset used in `image :` inside of the [public folder](https://github.com/tembo-io/website/tree/main/public)
- [x] Please write a `description: ` in the frontmatter of any new blog or doc
- [x] Read more [here](https://github.com/tembo-io/website?tab=readme-ov-file#writing-a-blog-post-%EF%B8%8F)

### CODE REVIEW

**Review by FloorD — COMMENTED** [review]
please also look at consistent punctuation, but this is great!

File: src/content/blog/2024-09-04-pgxn-status-update/index.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
+---
+slug: pgxn-v2-update
+title: What’s New on the PGXN v2 Project
+authors: [theory]
```
[the remark this answers is not in the document]
Comment (theory) [reply src/content/blog/2024-09-04-pgxn-status-update/index.md:4]: No? "on" and "the" are less important words. Want them all capitalized?

File: src/content/blog/2024-09-04-pgxn-status-update/index.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+tags: [pgxn, extensions, trunk]
```
[the remark this answers is not in the document]
Comment (theory) [reply src/content/blog/2024-09-04-pgxn-status-update/index.md]: Added in b33896d.

File: src/content/blog/2024-09-04-pgxn-status-update/index.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+# image: './pg-14-16.png'
+date: 2024-09-05T14:00
+description: |
+  A lot has happened in he five months since the last PGXN v2 update. The time
+  has been used for continuing community discussions, planning, designs, and the
+  start of implementation. Read on for a full accounting.
+---
```
[the remark this answers is not in the document]
Comment (theory) [reply src/content/blog/2024-09-04-pgxn-status-update/index.md:13]: Added in b33896d.

**Comment by theory** [comment]
Pushed another edit pass in 2e116d3. No real punctuation inconsistencies that I noticed. Happy to fix those I've overlooked!

**Review by DarrenBaldwin07 — APPROVED** [review]

**Review by FloorD — APPROVED** [review]

**Pull request merged**